qutebrowser/keyinput/keyutils.py

```diff
@@ -183,19 +136,79 @@ def _key_to_string(key):
         # For some keys, we just want a different name
         'Escape': 'Escape',
     }
-    # We now build our real special_names dict from the string mapping above.
-    # The reason we don't do this directly is that certain Qt versions don't
-    # have all the keys, so we want to ignore AttributeErrors.
-    special_names = {}
+
     for k, v in special_names_str.items():
         try:
-            special_names[getattr(Qt, 'Key_' + k)] = v
+            _SPECIAL_NAMES[getattr(Qt, 'Key_' + k)] = v
         except AttributeError:
             pass
-        special_names[0x0] = 'nil'
+        _SPECIAL_NAMES[0x0] = 'nil'
```

Comment:
Did anyone notice that this line should be outside of the for loop?

By the way: `_SPECIAL_NAMES = _build_special_names()` (with appropriately-defined `_build_special_names` may be cleaner.

(sorry for late comment.)